internal/controller/pipelinerun_helper.go

```diff
@@ -34,12 +36,12 @@ func AddFinalizerToPipelineRun(ctx context.Context, pipelineRun *tektonv1.Pipeli
 // RemoveFinalizerFromPipelineRun removes the finalizer from the PipelineRun.
 // If finalizer was not removed successfully, a non-nil error is returned.
 func RemoveFinalizerFromPipelineRun(ctx context.Context, pipelineRun *tektonv1.PipelineRun, r *NotificationServiceReconciler, finalizer string) error {
-	r.Log.Info("Removing finalizer")
+	r.Log.Info("Removing finalizer from %s", pipelineRun.Name)
 	patch := client.MergeFrom(pipelineRun.DeepCopy())
 	if ok := controllerutil.RemoveFinalizer(pipelineRun, finalizer); ok {
 		err := r.Client.Patch(ctx, pipelineRun, patch)
 		if err != nil {
-			return fmt.Errorf("Error occurred while patching the updated PipelineRun after finalizer removal: %w", err)
+			return fmt.Errorf("error occurred while patching the updated PipelineRun after finalizer removal: %w", err)
```

Comment:
There are a multiple changes not described in the commit message.

Reply:
I got warnings from `go-staticcheck`:
`error strings should not be capitalized (ST1005)go-staticcheck`
I had to remove the capitalization of these messages.